src/lib/dom.generated.d.ts

```diff
@@ -450,6 +540,18 @@ interface ElementDefinitionOptions {
     extends?: string;
 }
 
+interface EncodedAudioChunkInit {
+    data: AllowSharedBufferSource;
+    duration?: number;
```

Comment:
Maybe this is generically wrong with the DOM types, but this is seemingly an options bag given to a constructor. Why don't all of these have `| undefined`?

Reply:
I'd have to go read the code but I'm pretty sure it's an oversight that the generator never got updated for the missing/undefined distinction.